content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/count_products.rst

```diff
@@ -8,80 +8,95 @@ due to damages, human errors, theft, or other factors. As such, inventory adjust
    :alt: In stock products listed on the Inventory adjustments page.
 
-On the :guilabel:`Inventory Adjustments` page, all products that are currently in stock are listed,
+In the :guilabel:`Inventory Adjustments` table, all products that are currently in stock are listed,
 with each product line containing the following information:
 
-- :guilabel:`Location`: the specific location in the warehouse where a product is stored.
+- :guilabel:`Location`: the specific location in the warehouse where a product is stored. This field
+  needs to be enabled my going to :menuselection:`Inventory app --> Configuration --> Settings` and
+  ticking the :guilabel:`Storage Locations` checkbox.
 - :guilabel:`Product`: the product whose quantity is listed on the inventory adjustment line.
 - :guilabel:`Lot/Serial Number`: the tracking identifier assigned to the specific product listed. It
   can contain letters, numbers, or a combination of both.
-
-.. note::
-   If a specific product has a quantity of more than `1.00` in stock, and more than one serial
-   number (or lot number) assigned to it, each uniquely-identified product is displayed on its own
-   product line with its own lot/serial number, displayed under the :guilabel:`Lot/Serial Number`
-   column.
-
-- :guilabel:`On Hand Quantity`: the quantity of the product currently recorded in the database.
 - :guilabel:`UoM`: the *unit of measure* in which the product is measured. Unless otherwise
   specified (i.e., in :guilabel:`Pounds` or :guilabel:`Ounces`), the default :abbr:`UoM (Unit of
   Measure)` is :guilabel:`Units`.
 - :guilabel:`Counted Quantity`: the real quantity counted during an inventory count. This field is
-  left blank by default, but can be changed, depending on if it matches the :guilabel:`On Hand
+  left blank by default but can be changed, depending on if it matches the :guilabel:`On Hand
   Quantity` or not.
 - :guilabel:`Difference`: the difference between the :guilabel:`On Hand Quantity` and
   :guilabel:`Counted Quantity`, once an inventory adjustment is made. The difference is
   automatically calculated after every inventory adjustment.
-- :guilabel:`Scheduled Date`: the date at which a count should be made. If not otherwise specified,
-  this date will default to the 31st of December of the current year.
-- :guilabel:`User`: the person assigned to the count in the database. This can either be the person
-  physically counting the inventory, or applying the count in the database.
+
+.. note::
+   If a specific product has a quantity of more than `1.00` in stock, and more than one serial
+   number (or lot number) is assigned, each uniquely-identified product is displayed on its own
+   product line with its own lot/serial number, under the :guilabel:`Lot/Serial Number`
+   column.
+
+The fields below are optional and can be made visible by clicking the :icon:`oi-settings-adjust`
+:guilabel:`(settings)` icon in the top right of the :guilabel:`Inventory Adjustments` page.
```

Comment:
Nitpick, the hyphenated top-right is preferred :)
```suggestion
:guilabel:`(settings)` icon in the top-right of the :guilabel:`Inventory Adjustments` page.
```

Reply:
Since it's not modifying a noun, it's just top right in this case! :) (Zac and I hashed this one out already lol)